access.lua

```diff
@@ -198,13 +198,40 @@ then
     end
 end
 
+--
+-- 2 ... continued : portal assets that are available on every domains
+--
+-- For example: `https://whatever.org/ynhpanel.js` will serve the
+-- `/yunohost/sso/assets/js/ynhpanel.js` file.
+--
+
+if is_logged_in then
+    assets = {}
+    assets["/ynh_portal.js"] = "js/ynh_portal.js"
+    assets["/ynh_overlay.css"] = "css/ynh_overlay.css"
```

Comment:
According to https://stackoverflow.com/questions/513239/how-to-quickly-initialise-an-associative-table-in-lua
```suggestion
    assets = {
                   ["/ynh_portal.js"] = "js/ynh_portal.js",
                   ["/ynh_overlay.css"] = "css/ynh_overlay.css"
             }
```